packages/contracts-bedrock/test/dispute/FaultDisputeGameN.t.sol

```diff
@@ -486,34 +486,15 @@ contract FaultDisputeGameN_Test is FaultDisputeGame_Init {
         uint64 halfGameDuration = gameProxy.maxClockDuration().raw();
         uint64 clockExtension = gameProxy.clockExtension().raw();
 
-        // Make an initial attack against the root claim with 1 second left on the clock. The grandchild should be
-        // allocated exactly `clockExtension` seconds remaining on their potential clock.
+        // The move above the split depth's grand child is the execution trace bisection root. The grandchild should
```

Comment:
"The move above the split depth's grand child" means the move is located at `split depth's grand child` (`rootclaim`), but your meaning seems to be the move's grand child is the `rootclaim`. To be more clear, I would suggest adding comment like this "This specific move, located above the split depth, has a grandchild that acts as the multi-section root in the execution trace. Consequently, if the current move has less than `clockExtension` seconds remaining on its potential clock, the grandchild move should be allocated `clockExtension` * 2 seconds."